samples/pyodide-fastapi/config.capnp

```diff
@@ -23,7 +23,8 @@ const config :Workerd.Config = (
 const mainWorker :Workerd.Worker = (
   modules = [
     (name = "worker", pythonModule = embed "./worker.py"),
-    (name = "fastapi", pythonRequirement = "fastapi"),
+    (name = "fastapi==0.103.2", pythonRequirement = ""),
```

Comment:
will merge this, but I don't think we want to support version numbers and operators in here